CHANGELOG.md

```
@@ -1,4 +1,7 @@
 # React Native Kommunicate Chat v2.5.6
+- [Android] Fixed Assignee names are not updating if clientConversationId is set in buildConversation.
+- [iOS] Fixed SPM Build Issue.
+# React Native Kommunicate Chat v2.5.6
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)